desktop/cmp/error/ErrorMessage.ts

```diff
@@ -5,15 +5,15 @@
  * Copyright © 2023 Extremely Heavy Industries Inc.
  */
 import {div, frame, p} from '@xh/hoist/cmp/layout';
-import {hoistCmp, HoistProps, PlainObject} from '@xh/hoist/core';
+import {hoistCmp, HoistProps, PlainObject, TestSupportProps} from '@xh/hoist/core';
 import {button, ButtonProps} from '@xh/hoist/desktop/cmp/button';
 import '@xh/hoist/desktop/register';
 import {isEmpty, isNil, isString} from 'lodash';
-import {isValidElement, ReactNode, MouseEvent} from 'react';
+import {isValidElement, MouseEvent, ReactNode} from 'react';
 
 import './ErrorMessage.scss';
 
-export interface ErrorMessageProps extends HoistProps {
+export interface ErrorMessageProps extends HoistProps, TestSupportProps {
```

Comment:
Here is a case where upgrading this comp to support `BoxProps` should be straightforward and potentially useful - I would vote for going ahead with that, as we strip back other comps that said they support BoxProps but actually don't.